atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/SweepableCellFilterTest.java

```diff
@@ -139,4 +182,16 @@ public void thorough_getTimestampsToSweep_oneTransaction_emptyValue_returnsIt()
                         HIGH_START_TS, HIGH_COMMIT_TS));
         return ret;
     }
+
+    private List<CandidateCellForSweeping> setup(boolean lastIsTombstone, long lastCommitTs) {
+        Map<Long, Long> startTsToCommitTs = new HashMap<>(DEFAULT_COMMITS);
+        startTsToCommitTs.put(LAST_TS, lastCommitTs);
+        when(mockTransactionService.get(anyCollection())).thenReturn(startTsToCommitTs);
+        return ImmutableList.of(
+                ImmutableCandidateCellForSweeping.builder()
+                        .cell(SINGLE_CELL)
+                        .sortedTimestamps(startTsToCommitTs.keySet().stream().sorted().collect(Collectors.toList()))
```

Comment:
Streams are a bit overkill 😅 I'd probably write this as `ImmutableList.sortedCopyOf(startTsToCommitTs.keySet())` (non-blocking of course, just a style thing)